2022/src/day01.scala

```diff
@@ -0,0 +1,38 @@
+package day01
+
+import scala.util.Using
+import scala.io.Source
+import scala.math.Ordering
+
+import locations.Directory.currentDir
+
+@main def part1: Unit =
+  val input = Using.resource(Source.fromFile(s"$currentDir/../input/day01"))(_.mkString)
+  val answer = part1(input)
+  println(s"The solution is $answer")
+
+@main def part2: Unit =
+  val input = Using.resource(Source.fromFile(s"$currentDir/../input/day01"))(_.mkString)
```

Comment:
Should we create some `utilities.scala` file to automate this?

Reply:
Oh actually you kind of already started, with this very cool `currentDir`

Reply:
Also in case you're not aware, you can stick these in a `resources` dir and basically do the following:

```
//> using resourceDir "./resources"

import scala.io.Source

val input = Source.fromResource(fileName)
```

Reply:
added another one now for reading files synchronously

Reply:
@ckipp01 I wanted to do that, but then its not very portable if we do a scala.js solution